use_case_examples/hybrid_model/compile_hybrid_llm.py

```diff
@@ -0,0 +1,119 @@
+"""Showcase for the hybrid model converter."""
+
+import json
+import os
+from copy import deepcopy
+from pathlib import Path
+from typing import List, Union
+
+import torch
+from concrete.fhe import Configuration, ParameterSelectionStrategy
+from datasets import load_dataset
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from concrete.ml.torch.hybrid_model import HybridFHEModel
+
+
+def compile_model(
+    model_name: str,
+    model: torch.nn.Module,
+    inputs: torch.Tensor,
+    module_names: Union[str, List],
+    models_dir: Path,
+):
+    """Run the test for any model with its private module names."""
+
+    # Enable multi params/precision
+    configuration = Configuration(
+        single_precision=False,
+        parameter_selection_strategy=ParameterSelectionStrategy.MULTI,
+    )
+
+    # Create a hybrid model
+    hybrid_model = HybridFHEModel(model, module_names)
+    hybrid_model.compile_model(
+        inputs,
+        n_bits=8,
+        # We only do linear layers so no need to round
+        rounding_threshold_bits=None,
```

Comment:
Same here, I don't think any of our users is going to understand what this comment means. Let's just remove it as well as the rounding_bit_threshold perhaps?

Reply:
Well it's documented no?

I can remove if you prefer yes

Reply:
I am not sure. Let's keep it we will see.